<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadedTcpDemoClass</class>
 <widget class="QMainWindow" name="ThreadedTcpDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThreadedTcpDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="lstMsg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStart">
   <property name="icon">
    <iconset resource="ThreadedTcpDemo.qrc">
     <normaloff>:/ThreadedTcpDemo/Resources/MYSL.ICO</normaloff>:/ThreadedTcpDemo/Resources/MYSL.ICO</iconset>
   </property>
   <property name="text">
    <string>开始监听</string>
   </property>
  </action>
  <action name="actStop">
   <property name="icon">
    <iconset resource="ThreadedTcpDemo.qrc">
     <normaloff>:/ThreadedTcpDemo/Resources/APPTL.ICO</normaloff>:/ThreadedTcpDemo/Resources/APPTL.ICO</iconset>
   </property>
   <property name="text">
    <string>停止监听</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="ThreadedTcpDemo.qrc">
     <normaloff>:/ThreadedTcpDemo/Resources/OOFL.ICO</normaloff>:/ThreadedTcpDemo/Resources/OOFL.ICO</iconset>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ThreadedTcpDemo.qrc"/>
 </resources>
 <connections/>
</ui>
